Daily build features: describe optional CSV logging (#234)
mavlink/qgc-user-guide: en/SettingsView/csv.md

# CSV Logging

> **Note** This is a daily build feature (not present in QGC v3.5)

![Csv checkbox](../../assets/settings/general/csv.jpg)

> By checking this option, whenever a vehicle is connected and armed (unless "Save logs even if vehicle was not armed" is checked) a CSV (Comma-separated values) file will be created alongside the usual .tlog telemetry file.

Reviewer: Propose:

> When checked, a CSV (comma-separated value) telemetry file will be created along with the usual **.tlog** telemetry file.
  The file is only created if **Save log after each flight** is enabled, and is recorded for the same duration.

Is the option (and one above) disabled if the first **Save log after each flight** is disabled? If not, it should be!

Author: Yes it is. I just updated the text with your suggestions

> This CSV file contains the data from the vehicle Facts, which includes most of the relevant flight data quick analysis such as position, attitude, battery status, and others.

Reviewer 2: ```suggestion
This CSV file contains the most relevant vehicle telemetry data available for quick analysis such as GPS position, attitude, battery status, and others.
```

Reviewer: @Williangalvani I don't have permission to push to this repo for some reason. But I think you should accept this.
